sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -137,6 +137,17 @@ public void testDataSourceConfigurationDriverAndUrl() throws Exception {
   }
 
   @Test
+  public void testDataSourceConfigurationUsername() throws Exception {
+    JdbcIO.DataSourceConfiguration config = JdbcIO.DataSourceConfiguration.create(
+        "org.apache.derby.jdbc.ClientDriver",
+        "jdbc:derby://localhost:1527/target/beam",
+        "sa", "sa");
```

Comment:
Did this test fail without the current PR? If yes, can you amend the PR description to explain why the test was failing before / if no, can you add a different test that fails without the change and passes with it?